linode/manager
# M3-5122 create cluster UI updates

## Description

Updates the Create Cluster page to new UI spec and sets the default number of nodes to 3 in the Node Pool Panel. 

Copy and Kubernetes documentation link for less than 3 nodes needs to be updated and finalized. 

## How to test

Visit `/kubernetes/create` to view the UI changes and ensure pool selection still works as expected.
